modules/core/src/test/java/org/apache/ignite/internal/hlc/HybridTimestampTest.java

```diff
@@ -64,4 +67,62 @@ void notEqualWhenLogicalIsDifferent() {
     void hashCodeSameWhenComponentsAreSame() {
         assertEquals(new HybridTimestamp(1, 2).hashCode(), new HybridTimestamp(1, 2).hashCode());
     }
+
+    @Test
+    void addPhysicalTimeIncrementsPhysicalComponent() {
+        HybridTimestamp before = new HybridTimestamp(1, 2);
+
+        HybridTimestamp after = before.addPhysicalTime(1000);
+
+        assertThat(after.getPhysical(), is(1001L));
+    }
+
+    @Test
+    void addPhysicalTimeLeavesLogicalComponentIntact() {
```

Comment:
The point is to make unrelated properties be tested by separate tests, to make it easier to see what is being tested and what fails (if it does). Combining tests for a few properties in one test method makes it less obvious and only makes sense if each test scenario is rather heavy.

Reply:
I know why you did it, but in this case it's an unnecessary over complication, I think. You add some copy-pasted code just to check subtraction of two numbers, it won't make the other tests any more "heavy". It's up to you, of course, because it's obviously not a big deal